packages/open-payments/src/types.ts

```diff
@@ -1,4 +1,7 @@
-import { components } from './generated/types'
+import { components, paths as Paths } from './generated/types'
+
+export const getPath = <P extends keyof Paths>(path: P): string =>
```

Comment:
this function gives autocomplete functionality to all of the possible (generated) Open payments API paths we have